src/implementations/twig/components/social-media-follow/social-media-follow.html.twig

```diff
@@ -63,26 +48,56 @@
 
 <div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
   <p class="ecl-social-media-follow__description">{{ _description }}</p>
-{% if _links is not empty %}
-  <ul class="ecl-social-media-follow__list">
-  {% for link in _links %}
-  {% if link.label is defined and link.label is not empty %}
-    {% set _link_extra_classes = 'ecl-social-media-follow__link' %}
-    {% if link.extra_classes is not empty %}
-      {% set _link_extra_classes = _link_extra_classes ~ ' ' ~ link.extra_classes %}
+  {% if _links is not empty %}
+    <ul class="ecl-social-media-follow__list">
+    {% for link in _links %}
+      {# Backward compatibility provided by default(link) #}
+      <li class="ecl-social-media-follow__item">
+        {% set _link_extra_classes = 'ecl-social-media-follow__link' %}
+        {% if link.extra_classes is not empty %}
+          {% set _link_extra_classes = _link_extra_classes ~ ' ' ~ link.extra_classes %}
+        {% endif %}
+        {% set _icon_extra_classes = 'ecl-social-media-follow__icon' %}
+        {% if link.icon is not empty and link.icon.extra_classes is not empty %}
+          {% set _icon_extra_classes = _icon_extra_classes ~ ' ' ~ link.icon.extra_classes %}
+        {% endif %}
+        {% include '@ecl/link/link.html.twig' with link|merge({
```

Comment:
isn't this simply:
```
        {% include '@ecl/link/link.html.twig' with {
          link: link.link|default(link)|merge({
            type: 'standalone',
          }),
```

Reply:
not exactly. We also add a default class in extra_classes, while keeping the possiblity to have custom ones coming from the data (lines 54 and 60)

Reply:
sure, that was only a "partial", the full include would be:
```
        {% include '@ecl/link/link.html.twig' with {
          link: link.link|default(link)|merge({
            type: 'standalone',
          }),
          icon: link.icon|default({})|merge({
            extra_classes: _icon_extra_classes,
          }),
          extra_classes: _link_extra_classes,
        }) only %}
```
currently we have a nested merge, that is what i'm removing